comment_on_staging_test_change.sh

```diff
@@ -0,0 +1,26 @@
+#! /bin/sh
```

Comment:
better use `#!/bin/sh -e` to not stumble over unhandled, silent errors :)

Also it seems like you are actually using bash syntax, e.g. for arrays, not POSIX sh